inspect_image.go

```diff
@@ -93,7 +96,16 @@ func (c *Client) InspectImage(name string, daemon bool) (*ImageInfo, error) {
 			defaultProcessType = defaultProcess
 		}
 	} else {
-		defaultProcessType = defaultProcess
+		inspect, _, err := c.docker.ImageInspectWithRaw(context.TODO(), name)
+		if err != nil {
+			return nil, errors.Wrap(err, "reading image")
+		}
+
+		entrypoint := inspect.Config.Entrypoint
+		if len(entrypoint) > 0 && entrypoint[0] != LauncherEntrypoint {
+			process := strings.Replace(entrypoint[0], BpEntrypointPrefix, "", 1)
```

Comment:
On Windows, we might also need to strip `.exe`

Reply:
👍